config/settings.yml

```diff
@@ -1,3 +1,30 @@
+# the first endpoint_type listed must be for an local "online"
+# endpoint, and the default values listed for it will be used
+# in combination with the defined storage_roots to seed the db
+# with endpoints.
+endpoint_types:
+  local_online_nfs:
+    endpoint_class: online
+    endpoint_defaults:
+      endpoint_node: 'localhost'
+      recovery_cost: 1
+
 moab:
-  storage_roots: 'spec/fixtures'
+  # this is a list of named storage roots.  the names are used to seed the endpoints table.
+  storage_roots:
+    # name: '/location/on/disk'  # e.g.
+    fixtures: 'spec/fixtures'
```

Comment:
what was the reason for adding ```fixtures``` under ```storage_roots```? Is it because we have multiple service disks, so when running ```storage_roots(Settings.moab.storage_roots.map { |_storage_root_name, storage_root_location| storage_root_location })``` you can iterate through all of the service disks?

Reply:
so, the idea was just to give each storage root a name, and `fixtures` was the name i chose for the `fixtures` directory.  but in practice, it'd probably be more like:
```yml
storage_roots:
  services-disk08: '/services-disk08'
  services-disk09: '/services-disk09'
```

i'd thought about doing something more clever, like using the path or parsing something from the path, but this felt more flexible, in case the paths were ever not just a single directory level.